lib/ui/pages/account-linking/associated_accounts.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/associated_accounts_controller.dart';
+import 'package:pispapp/controllers/flow/account_linking_flow_controller.dart';
+import 'package:pispapp/models/consent.dart';
+import 'package:pispapp/models/currency.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/moja_button.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+
+// ignore: must_be_immutable
```

Comment:
non-immutable `StatelessWidget`? I think you can make this object non-immutable right

Reply:
Actually, I previously couldn't because I was using the constructor to set one of the fields but now I can using initializer lists :)